epictrack-api/manage.py

```diff
@@ -24,7 +24,7 @@
 from flask.cli import FlaskGroup
 
 
-APP = create_app(run_mode='migration')
+APP = create_app()
```

Comment:
I think we don't need to remove the 'run_mode'. I know that you did this so that you can run the application.right?

Reply:
I will revert this, I can always just remove it temporarily locally to run it